purchases/src/main/kotlin/com/revenuecat/purchases/paywalls/PaywallData.kt

```diff
@@ -44,6 +70,10 @@ data class PaywallData(
 
     @SerialName("localized_strings_by_tier") internal val localizationByTier:
     Map<String, Map<String, LocalizedConfiguration>> = emptyMap(),
+
+    @SerialName("zero_decimal_place_countries")
+    @Serializable(with = GoogleListSerializer::class)
+    val zeroDecimalPlaceCountries: List<String> = emptyList(),
```

Comment:
Ya, it's used to decode `google` only, and then flatten that directly onto `zeroDecimalPlaceCountries`, so we access that directly rather than going `zeroDecimalPlaceCountries.google` as it would be if it reflected the json.

I'm a bit hesitant to diverge from the json structure, however in this case it seems like a safe bet and makes the API nicer. Although thinking more now, it's possible that other Android stores could use different storefront codes...? If that were the case we'd have to choose the correct one to decode and expose via `zeroDecimalPlaceCountries`. I don't _think_ we'd ever need to have more than one set?

Reply:
Yea we could define the `ZeroDecimalPlaceCountries` object and omit the `apple` field (so it only has `google`). That should work because we have set `ignoreUnknownKeys = true` afaik. Then you'd indeed need to do `zeroDecimalPlaceCountries.google` but you avoid the need for a custom serializer. 

I'm fine either way. Not a blocker.